### ISSUE TITLE
Pages lack `<h1>` tags

### ISSUE THREAD

**Issue by zackkrida** [creativecommons/cccatalog-frontend#1069]
**Note: This is currently blocked by https://github.com/creativecommons/vocabulary/issues/529**.

The vast majority of our pages do not use `<h1>`tags for the primary title. This is terrible for a11y and also SEO.

- Homepage ("Search for content to reuse" is `<h2>`): https://search.creativecommons.org/
- Single Result (Image name is `<h5>`): https://search.creativecommons.org/photos/0cdd50d3-e8ca-41a6-ab8e-c71f48dc8807
- About (`<h2>`): https://search.creativecommons.org/about
- Search Syntax (`<h2>`): https://search.creativecommons.org/search-help

In fact, there are only 2 h1 tags in the entire codebase 😿 

<img width="372" alt="Screen Shot 2020-07-21 at 8 56 06 AM" src="https://user-images.githubusercontent.com/6351754/88057409-0a4e6500-cb30-11ea-94ed-f0e29306685f.png">

**Comment by zackkrida** [comment]
This is currently blocked by https://github.com/creativecommons/vocabulary/issues/529

**Comment by manueldevjour** [comment]
I can help with it if its possible.

**Comment by zackkrida** [comment]
Sure @manueldevjour but you will have to wait until _this_ issue is resolved: https://github.com/creativecommons/vocabulary/issues/529

**Comment by manueldevjour** [comment]
Cool @zackkrida hit me up at any moment for doing this

**Comment by zackkrida** [comment]
This will be unblocked next week by the latest update to https://github.com/creativecommons/vocabulary
the way to fix things will be going from `<h2>Lorem ipsum dolor set amet</h2>` to `<h1 class="title is-2">Lorem ipsum dolor set amet</h1>`

**Comment by chetas411** [comment]
> This will be unblocked next week by the latest update to https://github.com/creativecommons/vocabulary
> the way to fix things will be going from `<h2>Lorem ipsum dolor set amet</h2>` to `<h1 class="title is-2">Lorem ipsum dolor set amet</h1>`

So are these classes added in the code so that I can use them to fix this issue? @zackkrida

**Comment by zackkrida** [comment]
They will be added next week, but you can write your code with them now and I'll just wait to merge your code until next week!

**Comment by chetas411** [comment]
@zackkrida  can u review the PR now ?
